operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationService.java

```diff
@@ -364,4 +364,31 @@ default Set<Class<? extends HasMetadata>> defaultNonSSAResource() {
     return Set.of(ConfigMap.class, Secret.class);
   }
 
+  /**
+   * If a javaoperatorsdk.io/previous annotation should be used so that the operator sdk can detect events from its own updates
```

Comment:
```suggestion
   * If a {@code javaoperatorsdk.io/previous} annotation should be used so that the operator sdk can detect events from its own updates
```